pkg/util/metrics/helpers.go

```diff
@@ -18,8 +18,11 @@ package metrics
 import (
 	"context"
 	"fmt"
+	"strings"
 	"sync"
 
+	dto "github.com/prometheus/client_model/go"
```

Comment:
Nit: Why is it `dto`?

Reply:
lol good question, I don't know too.
I got this when looking at Prometheus lib source code: https://github.com/prometheus/common/blob/637d7c34db122e2d1a25d061423098663758d2d3/expfmt/text_parse.go#L25

Renamed it to pcm (Prometheus client model).